server/database/schemas/blue-ocean-schema.sql

```diff
@@ -7,6 +7,7 @@ DROP TABLE IF EXISTS notifications CASCADE;
 
 CREATE TABLE "users" (
   "id" serial PRIMARY KEY,
+  "uid" text UNIQUE NOT NULL,
```

Comment:
Needs to be VARCHAR(40) for better performance

Reply:
According to this: https://wiki.postgresql.org/wiki/Don't_Do_This#Text_storage
There's not a major performance difference between varchar and text and you should really only use varchar if you intentionally want the character limit. So you should go with text by default.